src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -141,10 +142,10 @@ export type AxisMenuSelection = {
     entrezGeneId?:number;
     genesetId?:string;
     treatmentId?:string;
-    selectedGeneOption?:{value:number, label:string}; // value is entrez id, label is hugo symbol
-    selectedDataSourceOption?:{value:string, label:string};
-    selectedGenesetOption?:{value:string, label:string};
-    selectedTreatmentOption?:{value:string, label:string};
+    selectedGeneOption?:PlotsTabGeneOption; // value is entrez id, label is hugo symbol
```

Comment:
nitpick but lets move the explanatory comment down to the type definition now (i.e. on line 166)